templates/tree-editor/example-file/example-file-command.ts

```diff
@@ -23,8 +24,14 @@ export class NewTreeExampleFileCommandHandler implements SingleUriCommandHandler
         protected readonly fileService: FileService,
         @inject(ILogger)
         protected readonly logger: ILogger,
+        @inject(WorkspaceService)
+        protected readonly workspaceService: WorkspaceService
```

Comment:
This reminds me that we should really refactor in the future to use property injection vs constructor injection:
- https://github.com/eclipse-theia/generator-theia-extension/issues/66

Reply:
I will comment on the issue #66